install-server.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -e
+
+echo -e "\n\nEverything is now getting setup. This process will take a few minutes...\n\n"
+
+sudo apt update && sudo apt install gpg jq -y && sudo apt clean
+
+#remove unattended upgrades
+sudo apt-get remove --purge unattended-upgrades || true
+
+#Install tailscale
+curl -fsSL https://tailscale.com/install.sh | sh
```

Comment:
should we manage this via nix as well?

Reply:
We can. I think it will take more work to keep tailscale updated. I noticed that with nixos, where it seemed tailscale was always outdated, compared to having debian manage it.